yarpc-core/src/main/java/io/yarpc/handler/TransportHandler.java

```
@@ -27,8 +27,6 @@
 
 import java.nio.ByteBuffer;
 
-public interface HandlerWrapper {
-    Response<ByteBuffer> handle(Request<ByteBuffer> transportRequest) throws RPCException;
-
+public interface TransportHandler extends Handler<ByteBuffer, ByteBuffer> {
     String getEncoding();
```

Doesn't that stop me from writing a generic catch-all handler that just deals with bytes and doesn't care what encoding is being used?